app/views/layouts/base.html.erb

```diff
@@ -41,8 +33,6 @@
       document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
     </script>
```

Comment:
Did we miss this tag manager bit?

Reply:
Oops - yes. https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1313